src/tool_getparam.c

```diff
@@ -493,6 +497,17 @@ static ParameterError GetSizeParameter(struct GlobalConfig *global,
   return PARAM_OK;
 }
 
+_Bool CheckFile(char *fname, _Bool override_file_prohibit)
```

Comment:
We use `bool` type for booleans and if this file is only used in this file it should also be `static`.